docs/_articles/en/user-guide/remote-development.md

```diff
@@ -1,21 +1,26 @@
 ---
-title: Remote Development - Containers
+title: Remote Development
 lang: en
 ---
 
 ## Overview
 
-Salesforce Extension for VS Code supports remote development and allows you to use a container as a full-featured development environment. You can open a project mounted into the container and edit with full code completions, code navigation, debugging, and more.
+Salesforce Extension for VS Code supports remote development and allows you to use a container as a full-featured development environment. You can open a project mounted into a Docker container or using WSL2 and edit with full code completions, code navigation, debugging, and more.
 
 > NOTICE: The remote development feature is currently in beta. If you find any bugs or have feedback, [open a GitHub issue](./bugs-and-feedback).
 
 Refer to VS Code documentation if you want to understand more about remote development:
 
 - Use a [Docker container](https://www.docker.com/) as your [development container](https://code.visualstudio.com/docs/remote/containers#_indepth-setting-up-a-folder-to-run-in-a-container)
+- Use [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/) as your [development container](https://code.visualstudio.com/docs/remote/wsl)
```

Comment:
Do we need to specify the "2" for WSL ?

Reply:
Yes, there is a previous version which is known as WSL or WSL 1